recipe/migrations/r410.yaml

```diff
@@ -0,0 +1,16 @@
+migrator_ts: 1621536224
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  automerge: true
+
+  # slow start, increase to 20/50 later
+  pr_limit: 2
+
```

Comment:
@isuruf should we use a key add migrator here and keep the bump number to zero?

Reply:
Key add migrator is more expensive because we'll be keeping 3.6.
Keeping bump number to zero is good

Reply:
Right. We may want to keep 3.6 for a while when this is running and then move it all at once.

Reply:
This should be a fairly quick migration (in contrast to a new Python version) keeping 3.6 shouldn't be necessary. We didn't do it the last time and I don't think we ran into any issues.

Reply:
OK. I will resolve this comment then.